.env.example

```diff
@@ -25,4 +25,8 @@ ELASTIC_PASSWORD=elastic_password
 ELASTIC_INDEX=elastic_index
 ELASTIC_TX_INDEX=elastic_tx_index
 ELASTIC_RESULTS_PER_PAGE=10
-ELASTIC_SEARCH_TIMEOUT=25
\ No newline at end of file
+ELASTIC_SEARCH_TIMEOUT=25
+WALLET_SERVICE_USERNAME=wallet_service_user
```

Comment:
minor: I would change this to `WALLET_SERVICE_DB_USERNAME` and `WALLET_SERVICE_PASSWORD` to `WALLET_SERVICE_DB_PASSWORD`

Reply:
Done, also changed `WALLET_SERVICE_HOST` to `WALLET_SERVICE_DB_HOST` and `WALLET_SERVICE_DB` to `WALLET_SERVICE_DB_NAME`